atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/CassandraTimeLockDialogueTestSuite.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.ete;
+
+import java.util.List;
+
+import org.junit.ClassRule;
+import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.atlasdb.containers.CassandraEnvironment;
+import com.palantir.atlasdb.ete.coordination.CoordinationEteTest;
+import com.palantir.atlasdb.ete.coordination.MultipleSchemaVersionsCoordinationEteTest;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+        TodoEteTest.class,
+        TargetedSweepEteTest.class,
+        CassandraTimestampsEteTest.class,
+        TimestampManagementEteTest.class,
+        CoordinationEteTest.class,
+        MultipleSchemaVersionsCoordinationEteTest.class,
+        LockWithTimelockEteTest.class
```

Comment:
I worry that lists of test suites like this will very quickly get out of sync with the c-j-r versions. Maybe OK if we can burn down the migration pretty fast, but i think if it takes longer than 2 weeks then this copy&pasting might be too much of a risk

Reply:
This is a valid concern - though the rate at which AtlasDB's ETE test classes change is incredibly slow (for instance, this class was last changed in February 2019) so I wouldn't be too worried.

Reply:
(And of course once the migration is done the non-Dialogue versions will go)